b64json/b64json.go

```diff
@@ -13,7 +13,19 @@ func Encode(v interface{}) (*string, error) {
 		return nil, err
 	}
 
-	b64Token := base64.StdEncoding.EncodeToString(jsonToken)
+	// Struct fields are marshalled in order of declaration, but we can't easily change the order
+	// Map fields are always marshalled alphabetically, so bounce through a map to force the order
+	var objmap map[string]interface{}
```

Comment:
Other implementations do this recursively, this one will only sort the first layer, won't it? I don't think we have nested structs so it may not be a problem.

Reply:
Yes. I some languages (like Java and Golang) I limit the scope because the expressiveness/YAGNI factor is not great.

Reply:
Yes. I some languages (like Java and Golang) I limit the scope because the expressiveness/YAGNI factor is not great.